theme-contract/src/main/java/com/orange/ouds/theme/OudsDrawableResources.kt

```diff
@@ -17,30 +17,68 @@ import com.orange.ouds.foundation.InternalOudsApi
 
 @InternalOudsApi
 interface OudsDrawableResources {
-    val alertImportant: Int
-        @DrawableRes get
-    val alertInformation: Int
-        @DrawableRes get
-    val alertSuccess: Int
-        @DrawableRes get
-    val alertWarningExternalShape: Int
-        @DrawableRes get
-    val alertWarningInternalShape: Int
-        @DrawableRes get
-    val checkboxIndeterminate: Int
-        @DrawableRes get
-    val checkboxSelected: Int
-        @DrawableRes get
-    val chipTick: Int
-        @DrawableRes get
-    val linkNext: Int
-        @DrawableRes get
-    val linkPrevious: Int
-        @DrawableRes get
-    val radioButtonSelected: Int
-        @DrawableRes get
-    val switchSelected: Int
-        @DrawableRes get
-    val tagClose: Int
-        @DrawableRes get
+    val component: Component
```

Comment:
We use plural for properties in `OudsTheme` (`borders`, `grids`, etc...), so I wonder if it should be `val component: Component` or `val components: Components`.